atoma-state/src/state_manager.rs

```diff
@@ -2160,23 +2162,17 @@ impl AtomaState {
     /// ```
     #[instrument(level = "trace", skip_all, fields(%small_id, %address))]
     pub async fn update_node_public_address(&self, small_id: i64, address: String) -> Result<()> {
-        sqlx::query(
-            "INSERT INTO node_public_addresses 
-                    (node_small_id, public_address) 
-                    VALUES ($1, $2) 
-                    ON CONFLICT (node_small_id)
-                    DO UPDATE SET public_address = EXCLUDED.public_address",
-        )
-        .bind(small_id)
-        .bind(address)
-        .execute(&self.db)
-        .await?;
+        sqlx::query("UPDATE nodes SET public_address = $2 WHERE node_small_id = $1")
```

Comment:
I do not understand how we can update a node address, isn't this its public key ?